assets/js/src/tracker/data-formatting.js

```diff
@@ -154,9 +156,9 @@ export const trackViewItem = ( {
 	listName = __( 'Product List', 'woocommerce-google-analytics-integration' ),
 } ) => {
 	if ( product ) {
-		trackEvent( 'view_item', {
+		return {
 			items: [ getProductImpressionObject( product, listName ) ],
-		} );
+		};
 	}
 };
 
```

Comment:
Some of the functions here still call `trackEvent`. Do I understand correctly, that it's because it's WIP, and eventually all functions here will only format/translate the data? Then the actual "tracking" will be covered by `tracker…track()`?

If so, maybe we could rename them from `track*` to `format*`?